fortran-src.cabal

```diff
@@ -0,0 +1,161 @@
+cabal-version: 1.12
```

Comment:
did you mean to commit a cabal file?

Reply:
Yes - I had been keeping them out before, but I changed my mind due to warnings (when using Git repos directly in Stack `extra-deps`) and glances at other packages doing it differently. I never grasped the general consensus on it - what do you usually do?

Reply:
Seems to be the recommended practice now. https://www.fpcomplete.com/blog/storing-generated-cabal-files/